package/piksi_system_daemon/overlay/etc/init.d/update_ntp_config

```diff
@@ -4,4 +4,5 @@ ntp_cfg="$1"; shift
 
 rm /etc/ntp.conf
 ln -sf "$ntp_cfg" /etc/ntp.conf
-monit restart ntpd
+/etc/init.d/S83endpoint_adapter_rpmsg_piksi100 stop
```

Comment:
Why are we killing the rpmsg adapter, but not restarting it?

Reply:
Good catch, copy/paste error - fixed!